components/ntp_sponsored_images/browser/ntp_sponsored_images_service_unittest.cc

```diff
@@ -69,11 +69,11 @@ TEST(NTPSponsoredImagesServiceTest, InternalDataTest) {
           "wallpapers": [
               {
                 "imageUrl": "background-1.jpg",
-                "focalPoint": {}
+                "focalPoint": { "x": 696, "y": 691 }
               },
               {
                 "imageUrl": "background-2.jpg",
-                "focalPoint": {}
+                "focalPoint": { "x": 1295, "y": 872 }
```

Comment:
we should make sure we test when the focalPoint value is missing, as it is optional

Reply:
Added test cased for it.